src/components/BrowserEventList/EventDescription/Selector.tsx

```diff
@@ -44,7 +44,7 @@ export function Selector({ selector, onHighlight }: SelectorProps) {
       onMouseEnter={handleMouseEnter}
       onMouseLeave={handleMouseLeave}
     >
-      {selector.css}
+      {selector.testId ?? selector.css}
```

Comment:
Not sure about the presentation. What do you think about something like this:

<img width="371" height="109" alt="Image" src="https://github.com/user-attachments/assets/298fe2c6-a3e9-4848-a22e-8192116fe42f" />